rosdep/python.yaml

```
@@ -6863,6 +6863,12 @@ python3-pyside2.qtopengl:
     '*': [python3-pyside2.qtopengl]
     bionic: null
     xenial: null
+python3-pyside2.qtuitools:
+  debian: [python3-pyside2.qtuitools]
```

@clalancette Can you confirm?
I think we should do something similar here, since it is not available in Debian Stretch or Jessie.
```suggestion
  debian:
    '*':  [python3-pyside2.qtuitools]
    stretch: null
    jessie: null
```

---

Both Debian Stretch and Jessie are out-of-support (see https://wiki.debian.org/LTS), so we don't need to add new entries for them.

---

I see. But Melodic is not EOL and it lists recommended support for Stretch: https://ros.org/reps/rep-0003.html#melodic-morenia-may-2018-may-2023

---

> I see. But Melodic is not EOL and it lists recommended support for Stretch: https://ros.org/reps/rep-0003.html#melodic-morenia-may-2018-may-2023

Ah, yeah, I can see where the confusion comes from.  I ended up disabling the Stretch jobs for Melodic back in December: https://discourse.ros.org/t/preparing-for-melodic-sync-2020-12-01/17681 .  I did not end up updating REP-3 along with that; maybe I should.

> If I understand the debian release cycle, Stretch is still supported by the LTS team.

Historically, we've only supported Debian (and Ubuntu, for that matter) until the end of the "normal" release cycle.  Typically the LTS beyond that requires setting up additional repositories and/or paying for support.  So from that perspective, Stretch is EOL.  But I'm not sure if this is documented anywhere.